### PR TITLE
terrain generation

### PR DESCRIPTION
**Pull request by Nwitz**

### CODE REVIEW

File: VS2017/Source/tests/TestLight.cpp [hunk 1]
```diff
@@ -29,6 +30,9 @@ namespace test {
 
 		GLCall(glEnable(GL_CULL_FACE));
 		glEnable(GL_DEPTH_TEST);
+
+		unsigned int uint = 5;
+		std::cout << uint / 2 << std::endl;
```
Comment (daniakalomiris) [inline VS2017/Source/tests/TestLight.cpp]: is there a purpose for this or was it just for testing?

**Review by daniakalomiris — APPROVED** [review]
Each terrain test generates a new random map - amazing work!

**Review by mairsarmy32 — APPROVED** [review]

**Review by danielvignypau — APPROVED** [review]
Looks good to me!

File: VS2017/Source/tests/TestLight.cpp (hunk 1 above)
Comment (Nwitz) [reply VS2017/Source/tests/TestLight.cpp]: Just testing, nice catch, removed

**Pull request merged**